app/javascript/miq-redux/store.js

```diff
@@ -7,6 +7,21 @@ const initialState = {};
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+const notificationInitialState = {
+  unreadCount: 0,
+};
+
+const nPrefix = '@@notifications/';
```

Comment:
It's just a semantic nitpick and others might disagree with me, but I'd prefer singular here. Just to be a little more future-prooof. Also the `setUnreadCount` is a little too specific and there's a 100% chance it will be dropped in the future in favor of something like `new`. So I'd just go with `@@notification/new` instead, but this is also something where others might and should stop me.

Reply:
Honestly i don't really care about this :D As long as its same everywhere I am ok with both.

Reply:
Yeah, in this case let's go with singular please.